package.json

```
@@ -5,18 +5,18 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "pnpm --recursive --stream run build",
-    "dev": "pnpm --recursive run dev",
-    "ui": "pnpm --filter @mosaic/ui run dev",
-    "test": "SKIP_INTEGRATION=true pnpm --recursive run test --passWithNoTests",
-    "test:integration": "pnpm --recursive run test:integration",
-    "test:all": "pnpm --recursive run test --passWithNoTests",
-    "lint": "pnpm --recursive run lint",
-    "lint:fix": "pnpm --recursive run lint:fix",
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "ui": "turbo dev --filter=@mosaic/app",
+    "test": "SKIP_INTEGRATION=true turbo test -- --passWithNoTests",
```

i'm down to rename this to `test:unit`if you think it's more clear

```suggestion
    "test:unit": "SKIP_INTEGRATION=true turbo test -- --passWithNoTests",
```